systest/unique_test.go

```diff
@@ -0,0 +1,583 @@
+//go:build integration
+
+/*
+ * Copyright 2017-2023 Dgraph Labs, Inc. and Contributors
+ *
```

Comment:
format the queries, name the test a bit better, improve the concurrency test, use dgraphtest